airbyte-integrations/connectors/source-mysql-strict-encrypt/metadata.yaml

```diff
@@ -11,7 +11,7 @@ data:
   connectorSubtype: database
   connectorType: source
   definitionId: 435bb9a5-7887-4809-aa58-28c27df0d7ad
-  dockerImageTag: 3.1.8
+  dockerImageTag: 3.1.10
```

Comment:
Why 3.1.10 instead of 3.1.9?

Reply:
3.1.9 was used by my previous PR (The one to extend debezium schema loading time)